cookieplone/settings.py

```diff
@@ -2,13 +2,18 @@
 #
 # SPDX-License-Identifier: MIT
 PLONE_MIN_VERSION = "6"
+PLONE_PYTHON_VERSIONS = {
+    (6, 0): ["3.8", "3.9", "3.10", "3.11", "3.12"],
```

Comment:
See both the current support grid, "Supported versions", and "Future versions" in https://plone.org/download/release-schedule.

@mauritsvanrees I also noticed that the Plone 6.1 section under "Future versions" needs to be updated to include Python 3.13.

```suggestion
    (6, 0): ["3.9", "3.10", "3.11", "3.12", "3.13"],
```